docs/requirements.txt

```diff
@@ -9,3 +9,4 @@ sphinx-autobuild
 sphinx_rtd_theme
 pydantic<2.0.0
 autodoc_pydantic
+olefile
```

Comment:
Why is this added to the docs requirements? Lets remove this

Reply:
> Why is this added to the docs requirements? Lets remove this

yep sorry for my mistake